pyfr/integrators/dual/controllers.py

```diff
@@ -41,22 +45,58 @@ class DualNoneController(BaseDualController):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
-        self._niters = self.cfg.getint('solver-time-integrator', 'niters')
+        sect = 'solver-time-integrator'
+
+        self._innersteps = 0
+        self._idxprev = 0
+        self._maxniters = self.cfg.getint(sect, 'niters')
+        self._atol = self.cfg.getfloat(sect, 'atol', default=1e-6)
```

Comment:
Probably want to rename the variables to something like `_iatol` and the config parameter as `inner-atol` so that we can be sure it will not clash with the regular atol.  Also not sure if we should have a default or not.